<?xml version="1.0"?>

<launch>
  <node name="pan_tilt_sensor_head_joint_control" pkg="hector_camera_joint_controller" type="cam_joint_trajectory_controller" respawn="true" output="screen">
    <param name="controller_namespace" value="/tracker_controllers/sensor_head_traj_controller" />
    <param name="control_loop_period" value="0.1" />
    <param name="robot_link_reference_frame" value="arm_link_4" />
  </node>
</launch>
